api_tests/preprint_providers/views/test_preprint_provider_subjects_list.py

```diff
@@ -6,6 +6,28 @@
 from osf_tests.factories import SubjectFactory, PreprintProviderFactory
 
 
+class TestPreprintProviderExists(ApiTestCase):
+
+    # Regression for https://openscience.atlassian.net/browse/OSF-7621
+
+    def setUp(self):
+        super(TestPreprintProviderExists, self).setUp()
+        self.url = '/{}/preprint_proiders/fake/'.format(API_BASE)
```

Comment:
Ah, nice catch. 

Gonna move into a new file as well. I didn't realize I added this test in `test_preprint_provider_subjects_list.py`, where it doesn't really belong.